Tasks/InstallAppleCertificate/preinstallcert.ts

```diff
@@ -31,7 +31,7 @@ async function run() {
             throw tl.loc('INVALID_P12');
         }
         tl.setTaskVariable('APPLE_CERTIFICATE_SHA1HASH', p12Hash);
-        tl.setVariable('APPLE_CERTIFICATE_SIGNING_IDENTITY', p12CN);
+        tl.setVariable('SigningIdentity', p12CN);
```

Comment:
Below in this file, we set APPLE_CERTIFICATE_KEYCHAIN, that will be useful to expose as an output variable too.

Reply:
Done.